### PR TITLE
Pass image store function to pana

### PR DESCRIPTION
**Pull request by szakarias**

### CODE REVIEW

File: app/lib/analyzer/pana_runner.dart [hunk 1]
```diff
@@ -72,18 +74,24 @@ class _PanaRunner implements PanaRunner {
...
+              version: version,
+              options: InspectOptions(
+                isInternal: isInternal,
+                pubHostedUrl: activeConfiguration.primaryApiUri.toString(),
+                analysisOptionsYaml: packageStatus.usesFlutter
+                    ? null
+                    : await getDefaultAnalysisOptionsYaml(),
+              ),
+              logger: Logger.detached('pana/$package/$version'),
+              storeResource: store);
```
Comment (isoos) [inline app/lib/analyzer/pana_runner.dart:94]: Nit: `storeResource: store,` (`,` at the end)

File: app/lib/scorecard/models.dart [hunk 2]
```diff
@@ -7,7 +7,13 @@ import 'dart:io';
 import 'package:clock/clock.dart';
 import 'package:json_annotation/json_annotation.dart';
 import 'package:pana/models.dart'
-    show LicenseFile, PanaRuntimeInfo, Report, ReportSection, UrlProblem;
+    show
```
Comment (isoos) [inline app/lib/scorecard/models.dart:10]: At this point we may just import it unconditionally?

**Review by isoos — APPROVED** [review]

File: app/lib/analyzer/pana_runner.dart (hunk 1 above)
Comment (szakarias) [reply app/lib/analyzer/pana_runner.dart]: Done

File: app/lib/scorecard/models.dart (hunk 2 above)
Comment (szakarias) [reply app/lib/scorecard/models.dart]: Done

File: app/test/analyzer/pana_runner_test.dart [hunk 3]
```diff
@@ -54,5 +55,63 @@ void main() {
...
     );
+
+    testWithProfile(
+      'screenshot test',
+      testProfile: TestProfile(
+        packages: [
+          TestPackage(
+              name: '_dummy_pkg', versions: [TestVersion(version: '0.0.193')]),
+        ],
+        defaultUser: 'admin@pub.dev',
```
Comment (jonasfj) [inline app/test/analyzer/pana_runner_test.dart:66]:
```suggestion
        importSource: ImportSource.fromPubDev(),
        defaultUser: 'admin@pub.dev',
```

**Review by jonasfj — APPROVED** [review]

**Review by jonasfj — APPROVED** [review]

**Pull request merged**